Practice/E_Shipunova/5.3.py

```diff
@@ -0,0 +1,44 @@
+import os
+#import tempfile - I can not use the library, because my interpreter trows Error already at import
```

Comment:
What error is thrown? What is interpreter version?

Reply:
Текст ошибки:
C:\Users\admin\AppData\Local\Programs\Python\Python38-32\python.exe C:/Users/admin/PycharmProjects/111/5.3.py
[(0, 9), (1, 8), (2, 7)]
Traceback (most recent call last):
  File "C:/Users/admin/PycharmProjects/111/5.3.py", line 2, in <module>
    import tempfile  #- I can not use the library, because my interpreter trows Error already at import
  File "C:\Users\admin\AppData\Local\Programs\Python\Python38-32\lib\tempfile.py", line 43, in <module>
    import shutil as _shutil
  File "C:\Users\admin\AppData\Local\Programs\Python\Python38-32\lib\shutil.py", line 10, in <module>
    import fnmatch
  File "C:\Users\admin\AppData\Local\Programs\Python\Python38-32\lib\fnmatch.py", line 14, in <module>
    import re
  File "C:\Users\admin\AppData\Local\Programs\Python\Python38-32\lib\re.py", line 145, in <module>
    class RegexFlag(enum.IntFlag):
AttributeError: module 'enum' has no attribute 'IntFlag'

Process finished with exit code 1
==========
Версия python: 3.8.4
==========
Поняла в чем дело: на одной из практик была задачка про реализацию ф-ции emumerate(), ф-л под эту задачку назвала enum.py, импортированной библиотеке tempfile понадобился модуль enum... ближайшим модулем с данным именем стала эта задачка, и разумеется никакого атрибута  'IntFlag' в ней найдено не было, вот и ошибка.
Ф-л удалила, импорт tempfile и использование mktemp() отработали успешно. Код 5.3 отредактирован.

Reply:
Отлично! Да, конфликты из-за неудачных имен файлов часто случаются. Отчасти могло бы помочь использование виртуального окружения для каждой задачи, но и это не панацея (и неудобно к тому же).